packages/block-editor/src/components/global-styles/test/get-global-styles-changes.js

```diff
@@ -1,9 +1,21 @@
 /**
  * Internal dependencies
  */
-import getRevisionChanges from '../get-revision-changes';
+import getGlobalStylesChanges from '../get-global-styles-changes';
 
-describe( 'getRevisionChanges', () => {
+jest.mock( '@wordpress/blocks', () => {
```

Comment:
Just a quick question while skimming through: why is this being mocked instead of calling `registerBlockType` to register a block for use in the tests? It looks like a lot of the current tests will register a test block and then [unregister](https://github.com/WordPress/gutenberg/blob/7358f66b08d1276d1bc919dd3f52be91020c0220/packages/block-editor/src/components/block-controls/test/index.js#L53) via:

```
	afterEach( () => {
		getBlockTypes().forEach( ( block ) => {
			unregisterBlockType( block.name );
		} );
	} );
```

So was just wondering if it's worth doing that here, too?

Reply:
Thanks for checking it out!

Do you see an advantage to registering/unregistering a block over a mock implementation? 

I don't have a preference, only to return the bare minimum required for the tests.

Reply:
This could just be a personal preference, but I like it when tests can use the "real" system as much as possible without having to mock or override behaviour in order to get tests passing. In that way, the tests follow the overall APIs in place rather than having to have something custom for a set of tests (so, perhaps I lean a little more to the integration-style tests over more unit-style tests). I see there are a few other tests using `jest.mock( '@wordpress/blocks'`, but most seem to be using `registerBlockType` as far as I can tell.

My main thought whenever I see mocks, though, is "Oh, I wonder why that's mocked, why does a separate API need to be overridden?", so it sticks out a little to me, but for these particular tests, I'm not sure it matters either way. Not a blocker for me, happy for us to go with what you think works best here! 🙂

Reply:
That's convincing enough for me and the outcome is the same. 😄 

Thanks for explaining. I will update.